build.sbt

```
@@ -57,7 +57,8 @@ val commonSettings = List(
     "co.fs2" %%% "fs2-core" % fs2Version,
     "org.scala-lang.modules" %%% "scala-collection-compat" % "2.7.0",
     "io.circe" %%% "circe-parser" % circeVersion % "test",
-    "co.fs2" %% "fs2-io" % fs2Version % "test",
+    "io.circe" %%% "circe-jawn" % "0.15.0-M1" % "test",
```

```suggestion
    "io.circe" %%% "circe-jawn" % circeVersion % "test",
```

Or is there any reason we need the milestone instead of the stable `0.14.1` that's used elsewhere?

---

Yes, to get the circe-jawn parser on JS. It will be available in 0.14.2 when that's released.

---

I’m fine with that since it’s for testing purpose.